### PR TITLE
Created Naughty Natural scraper

### PR DESCRIPTION
**Pull request by echo6ix**
This creates a scene and gallery scraper for naughtynatural.com

The scraper does not scrape a scene image. The xPath scene scraper correctly parses the scene's jpg file, however the server returns the following:

```
<Error>
<Code>AccessDenied</Code>
<Message>Access Denied</Message>
<RequestId>W708A9V1FYCYGYMP</RequestId>
<HostId>Zxrl4lZH6o9mKpN2kJeAaWPpigw21xFcNk9sH7TIkQdf2fZnOoOKzrGdZ/xdCIzFm+WybHr8img=</HostId>
</Error>
```

This is beyond my scope of expertise. Perhaps someone with more advanced skills can figure it out, if anyone even uses this scraper.

### CODE REVIEW

File: scrapers/NaughtyNatural.yml
Comment (Maista6969) [inline scrapers/NaughtyNatural.yml:40]: I noticed that while they are using AWS to store these images, they do also store it right on their CMS as well... we can do heinous regex stuff to grab it from there instead! If we grab the path from the first URL (the image) and the domain from the second URL (the video) and Frankenstein them together then it resolves and we can scrape the image

```
Image:
  selector: //script[@type="text/javascript"][contains(text(),"jwplayer") and contains(text(),"jpg")]
  postProcess:
    - replace:
        - regex: .*(wp-content.+jpg).*(https:\/\/[^\/]*).*
          with: $2/$1
```

**Review by Maista6969 — APPROVED** [review]
Looks good to me! I have a suggestion for how to grab the image so I'll wait for you to consider that and give the go-ahead before I merge this

File: scrapers/NaughtyNatural.yml
Comment (echo6ix) [reply scrapers/NaughtyNatural.yml]: Perfect. Done.

**Pull request merged**
